app/src/main/kotlin/com/github/premnirmal/ticker/settings/ExportWidget.kt

```diff
@@ -0,0 +1,32 @@
+package com.github.premnirmal.ticker.settings
+
+import com.github.premnirmal.ticker.AppPreferences.Companion.toCommaSeparatedString
+import com.github.premnirmal.ticker.widget.WidgetData
+
+class ExportWidget() {
+    lateinit var name: String
```

Comment:
it looks like the indentation is off in some of these files. Can you makes sure you follow the style guide listed in the contribution page https://github.com/premnirmal/StockTicker/blob/master/CONTRIBUTING.md#contributing-to-stockticker?

Reply:
Yeah sorry I was just after some early input

Reply:
I am unable to use that project, but that is unrelated to this issue, I will start another.

Reply:
I am using the latest version of Android Studio is that going to be an issue?
There seems to be no corelation between the project you linked to and code styles in android studio

Reply:
The instructions for the codestyle are in https://github.com/square/java-code-styles